pkg/zoom/message_types.go

```diff
@@ -10,6 +10,9 @@ import (
 WEBSOCKET MESSAGE TYPES
 */
 
+type Message interface {
+}
```

Comment:
this should have an `isMessage()` method, which you'd then add to each possible message type that would be handled by `onMessage`. I would have done it myself, but I wasn't sure which structs corresponded to top-level messages.